test/api/buffer/exports/delete/expected-etcd-kvs.txt

```diff
@@ -87,7 +87,7 @@
         "content": "body.key+\":\"+body.value"
       }
     ]
- },
+  },
```

Comment:
:slightly_smiling_face: 

https://www.diffchecker.com/i1doa7iJ

![image](https://user-images.githubusercontent.com/19371734/206007630-5186b499-7f14-474d-831f-72c9e90a2cb2.png)

---------------------

Samozrejme, z toho diffu v konzole sa to tazko hlada, ... otazka je, ci sa to tam vobec da lepsie zobrazit, ked je tam tolko `wildcards`.

Reply:
ómg, díky. 🙈 Kontroloval jsem to po řádcích side-by-side a ta mezera mi teda unikla. Tak příště diff toolem. 😅